.travis.yml

```diff
@@ -1,70 +0,0 @@
-language: python
```

Comment:
Could we also swap out the build status badge in `README.rst` for GitHub actions badge(s)?

https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

Reply:
Added in the next commit.